### PR TITLE
refactor: add rules for color hex length and color hex case && change…

### PR DESCRIPTION
**Pull request by Natalie-Smirnova**
- add rules for color hex length and color hex case
- change `color-hex-length` and `color-hex-case` in `.less` files

### CODE REVIEW

File: .stylelintrc.yml [hunk 1]
```diff
@@ -83,6 +83,10 @@ rules:
   unit-no-unknown: true
   # Disallow absolute paths for imports in bounds of library files
   esl-less/import-root: never
+  # Disallow long notation for hex colors
+  color-hex-length: short
+  # Disallow uppercase for hex colors
+  color-hex-case: lower
```
Comment (dshovchko) [inline .stylelintrc.yml:89]: I think it would be easier to understand if the rules were sorted alphabetically

**Review by dshovchko — COMMENTED** [review]
There is a problem:
sshot_2022-07-05-15-57-30

**Review by dshovchko — APPROVED** [review]

**Review by NastaLeo — APPROVED** [review]

**Pull request merged**